packages/lesswrong/lib/collections/users/helpers.tsx

```diff
@@ -200,21 +200,32 @@ export const userIsAllowedToComment = (user: UsersCurrent|DbUser|null, post: Pos
   if (user.allCommentingDisabled) return false
   if (user.commentingOnOtherUsersDisabled && post?.userId && (post.userId != user._id)) return false // this has to check for post.userId because that isn't consisently provided to CommentsNewForm components, which resulted in users failing to be able to comment on their own shortform post
 
-  if (!post) return true
-  if (post.commentsLocked) return false
-  if (post.rejected) return false 
-  if ((post.commentsLockedToAccountsCreatedAfter ?? new Date()) < user.createdAt) return false
-
-  if (userIsBannedFromPost(user, post, postAuthor)) {
-    return false
-  }
-
-  if (userIsBannedFromAllPosts(user, post, postAuthor)) {
-    return false
-  }
+  if (post) { {
```

Comment:
extra braces

Reply:
or, maybe not extra, but misplaced?

Reply:
Whoa, that was a weird editor mishap that I believe would make this pretty broken.